cupy_setup_build.py

```diff
@@ -83,6 +83,7 @@
     'cupy.cuda.function',
     'cupy.cuda.stream',
     'cupy.cuda.texture',
+    'cupy.cutensor',
```

Comment:
Could you move this under cutensor module?

Reply:
Fixed.